integration/cmd/osrm-traffic-updater/speed_table_dumper.go

```diff
@@ -109,6 +128,17 @@ func task(wayid2speed map[int64]int, source <-chan string, sink chan<- string, d
 	tasksWg.Done()
 }
 
+func getSpeedOfSegments(segments []*trafficproxy.SegmentedFlow, speeds []int, nodesCnt uint64) {
```

Comment:
1. Would be better to have some protection here:
```
0 <= segment.Begin < segment.End <=100
```
Ignore unexpected data and generate a warning.

2. How about following situation:
A long edge just has two nodes, and we generate a traffic flow segment
```
[speed:25 begin:25 end:75 ]
```
It will result 
```
node_1, node_1, 25
```
Shall we filter this situation: indexOfBegin == indexOfEnd?

Reply:
@CodeBear801 I've added the warning and info into statistics